gateway-server/src/main/java/org/apache/knox/gateway/websockets/ProxyWebSocketAdapter.java

```diff
@@ -313,4 +314,8 @@ private void closeQuietly() {
       frontendSession.close();
     }
   }
+
+  public void setMaxBufferCount(int count) {
+    this.maxMessageBufferCount = count;
+  }
```

Comment:
I'd remove this API completely; it's redundant.

Reply:
Removed. Thanks